shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -458,7 +458,14 @@ class JSBackend(allowUnresolvedSymbols: Boolean) {
     fields.foreach(constructorScope.declareValue(_, Some(false), false))
     val rest = constructorScope.declareValue("rest", Some(false), false)
     val base = getterScope.declareValue("base", Some(false), false)
-    val stmts = mixinSymbol.ctor.map(s => JSExprStmt(translateTerm(s)(constructorScope)))
+    val getter = new ListBuffer[Str]()
+    val stmts = mixinSymbol.ctor.map(s => s match {
```

Comment:
Actually, these two cases are all we need to handle here. Other cases would be handled in `members`, but it's always a good idea to add a default case.

Reply:
Note that your code is not supposed to emit warnings. The code base is currently not clean so it does have warnings, but please try to avoid adding any!